prog/dftb+/lib_elecsolvers/elsisolver.F90

```diff
@@ -345,7 +345,7 @@ subroutine TElsiSolver_init(this, inp, env, nBasisFn, nEl, iDistribFn, nSpin, iS
       this%spinDegeneracy = 0.0_dp
     end if
 
-    this%nSpin = nSpin
+    this%nSpin = min(nSpin, 2) ! copes with 2 component spin
```

Comment:
Why min?

Reply:
min(x,2) as occupation in ELSI is 2 for nSpin = 1 and 1 for nSpin = 2 (don't think it knows about nSpin = 4, but partly tests for > 1 in some places).